run.py

```diff
@@ -0,0 +1,9 @@
+import os
+from app import create_app
+
+config_name = os.getenv('APP_SETTINGS')
+app = create_app(config_name)
+
+
+if __name__ == '__main__':
+    app.run()
```

Comment:
This file is not required if you have `manage.py`. You can run the app by using manager command. i.e `python manage.py runserver` instead of `flask run`